app/service/data_svc.py

```diff
@@ -65,6 +65,27 @@ def __init__(self, dao):
                 for fact in source['facts']:
                     fact['source_id'] = source_id
                     await self.create_fact(**fact)
+                relationships = source.get('relationships')
+                if relationships:
+                    for relationship in relationships.values():
+                        await self.create_relationship(relationship)
+
+    async def create_relationship(self, values):
```

Comment:
move this function under the """ CREATE """ section in this module. also make the parameters explicit - have it accept 3 parameters (value1, relationship, value2). I like @wbooth's suggestion to rename these source and target as well. also, since this is a public function, add a docstring.